.github/scripts/ut.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env sh
```

Comment:
WDYT on renaming this file to `unit-test.sh`

Reply:
yep, can do.